test/WebJobs.Script.Tests/ScriptHostTests.cs

```diff
@@ -1031,6 +1031,56 @@ public void UpdateProxyName(string proxyName)
             Assert.Equal("myproxy", ProxyMetadataManager.NormalizeProxyName(proxyName));
         }
 
+        [Fact]
+        public void FunctionsSupported_Returns_True()
```

Comment:
nit: the method name here should be something like `IsSingleLanguage_WithSupportedLanguages_Returns_True`